### ISSUE TITLE
Does franca support `join` query?  how hard is it to implement this lib using ruby?

### ISSUE THREAD

**Issue by ruichao-factual** [chenguo/franca-js#12]
Sometimes, we need to fetch data that are stored in multi tables, especially for relational database.  Does `franca-js` support this kind of query?

Another question, is it hard to implement a similar lib using ruby?

**Comment by chen-factual** [comment]
Yes, not but directly. We specify a `table` field that goes into the SQL query, i.e. `SELECT * FROM table`.  So for the table field you can actually drop a join, like if you set `table: "tabA INNER JOIN tabB ON tabA.id = tabB.id", your select query will be`SELECT \* FROM tabA INNER JOIN tabB ON tabA.id = tabB.id`.

> Another question, is it hard to implement a similar lib using ruby?

All Franca does is translating a map to a string (Postgres, Solr) or another map (Mongo), so it's should be totally doable in Ruby. When Fanchen's done with re-architecting for writes, it shouldn't be difficult to do a Ruby port if that's desired (I'm guessing for rails?)
